protocol/protobuf/application_metadata_message.proto

```diff
@@ -97,5 +97,6 @@ message ApplicationMetadataMessage {
     SYNC_ACTIVITY_CENTER_COMMUNITY_REQUEST_DECISION = 77;
     SYNC_TOKEN_PREFERENCES = 78;
     COMMUNITY_PUBLIC_SHARD_INFO = 79;
+    SYNC_COMMUNITY_STORENODES = 80;
```

Comment:
We use the `SYNC` word in terms of syncing multiple Status instances of the same account. So it's not quite correct to use here. I'd suggest this, again, similar to community shard info:
```suggestion
    COMMUNITY_PUBLIC_STORENODES_INFO = 80;
```